lib/node_modules/@stdlib/repl/README.md

```diff
@@ -83,6 +84,28 @@ The function supports specifying the following settings:
 -   **autoDeletePairs**: boolean indicating whether to automatically delete adjacent matching brackets, parentheses, and quotes. Default: `true`.
 -   **autoPage**: boolean indicating whether to automatically page return values having a display size exceeding the visible screen. When streams are TTY, the default is `true`; otherwise, the default is `false`.
 -   **completionPreviews**: boolean indicating whether to display completion previews for auto-completion. When streams are TTY, the default is `true`; otherwise, the default is `false`.
+-   **syntaxHighlighting**: boolean indicating whether to enable syntax highlighting of the input. When streams are TTY, the default is `true`; otherwise, the default is `false`.
+-   **theme**: initial color theme for syntax highlighting. Default: `default`.
+-   **defaultTheme**: default theme for syntax highlighting. Default: `default`.
```

Comment:
Now looking at this, having `theme` AND `defaultTheme` doesn't seem necessary. The default theme should be whatever `theme` is set to. In which case, I think we can remove the `defaultTheme` option and setting. Sorry, I think I advocated for this originally, but I now consider this non-ideal.